.kokoro/docker/cloudbuild-test.yaml

```diff
@@ -0,0 +1,57 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+timeout: 7200s # 2 hours
+substitutions:
+  _JAVA_SHARED_CONFIG_VERSION: '1.5.7'
+  _GRAALVM_A: 'graalvm22.3_jdk11'
+  _GRAALVM_B: 'graalvm22.3_jdk17'
+steps:
+  # GraalVM build
+  - name: gcr.io/cloud-builders/docker
+    args: ["build", "-t", "gcr.io/$PROJECT_ID/graalvm_a:${_JAVA_SHARED_CONFIG_VERSION}", "."]
+    dir: .kokoro/docker/${_GRAALVM_A}
+    id: graalvm-build
+    waitFor: ["-"]
+  - name: gcr.io/gcp-runtimes/structure_test
+    args:
+      ["-i", "gcr.io/$PROJECT_ID/graalvm_a:${_JAVA_SHARED_CONFIG_VERSION}", "--config", ".kokoro/docker/${_GRAALVM_A}.yaml", "-v"]
+    waitFor: ["graalvm-build"]
+  - name: gcr.io/cloud-builders/docker
+    args:
+      [
+          "tag",
+          "gcr.io/$PROJECT_ID/graalvm_a:${_JAVA_SHARED_CONFIG_VERSION}",
+          "gcr.io/cloud-devrel-public-resources/graalvm_a:${_JAVA_SHARED_CONFIG_VERSION}",
+      ]
+    waitFor: ["graalvm-build"]
+
+  # GraalVM JDK 17 build
+  - name: gcr.io/cloud-builders/docker
+    args: ["build", "-t", "gcr.io/$PROJECT_ID/graalvm_b:${_JAVA_SHARED_CONFIG_VERSION}", "."]
+    dir: .kokoro/docker/${_GRAALVM_B}
+    id: graalvm17-build
```

Comment:
These IDs now assume knowledge of JDK version they shouldn't know. Change to graalvm-b-build to avoid it becoming desync'd.

Reply:
Good catch, done.